emails/verification.spt

```diff
@@ -1,16 +1,63 @@
 {{ _("Connect to {0} on Gratipay?", username) }}
 
 [---] text/html
-{{ _("We've received a request to connect {0} to the {1} account on Gratipay. Sound familiar?",
-     ('<b>%s</b>'|safe) % email,
-     ('<b><a href="https://gratipay.com/~{0}/">{0}</a></b>'|safe).format(username)) }}
+{% if new_email_verified %}
+{{ ngettext( "We've received a request to connect the {package_name} npm package to the "
+             "{username} account on Gratipay. Sound familiar?"
```

Comment:
Down on line 32 there's a "Yes, proceed!" button. Here's what it looks like:

![screen shot 2017-04-13 at 8 41 55 am](https://cloud.githubusercontent.com/assets/134455/25005013/19f99eb6-2025-11e7-8c5b-71e17939d08a.png)